### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by elenatvk**
doc review of the readme

### CODE REVIEW

File: templates/sag-db-sqlserver-103-and-lower/README.md [hunk 1]
```diff
@@ -50,28 +50,26 @@ sagcc exec templates composite apply sag-db-sqlserver-103-and-lower
...
 
-Launch the [Miscrosoft SQL Server on Linux for Docker Engine](https://hub.docker.com/r/microsoft/mssql-server-linux/) container:
+2. Launch the [Miscrosoft SQL Server on Linux for Docker Engine](https://hub.docker.com/r/microsoft/mssql-server-linux/) container:
 
 ```bash
 docker-compose -f templates/sag-db-sqlserver-103-and-lower/docker-compose.yml up -d sqlserver
 ```
 
-Provision sag-db-sqlserver template and run tests:
+3. Provision the `sag-db-sqlserver-103-and-loewr` template and run tests:
```
Comment (YanaSimeonova) [inline templates/sag-db-sqlserver-103-and-lower/README.md]: Elly, there is a typo (loewr)

File: templates/sag-db-sqlserver-103-and-lower/README.md [hunk 2]
```diff
@@ -50,28 +50,26 @@ sagcc exec templates composite apply sag-db-sqlserver-103-and-lower
...
 
 ```bash
 CC_ENV=sqlserver ./provisionw sag-db-sqlserver-103-and-lower
 ```
-
 If the test is successful, the test ouptut contains `TEST SUCCESSFUL`.
```
Comment (YanaSimeonova) [inline templates/sag-db-sqlserver-103-and-lower/README.md]: One more typo - ouptut

File: templates/sag-db-mysql/README.md [hunk 3]
```diff
@@ -59,27 +84,26 @@ When importing the composite template in Command Central, you have to attach the
...
+To install Database Component Configurator 10.5 on the Command Central node with alias `local`, create a database named `webm` and a database user named `webm` with password `webm`, and create database schemas for all products, except API Gateway, with user `root` and password `root`:
 
 ```bash
 sagcc exec templates composite apply sag-db-mysql \
   db.version=latest repo.product=products-10.5 repo.fix=fixes-10.5 nodes=local \
   db.host=mysql db.admin.username=root db.admin.password=root \
   db.name=webm db.username=webm db.password=webm \
-  db.MWS.components=[] db.STR.components=[STR]\
-  db.product.scripts=[DatabaseComponentConfigurator, OBEcdc, WOKcdc, TNScdc, PIEcdc, PIEEmbeddedCdc, PIEMobileCdc, WPEcdc] \
+  db.YAI.components=[] \
```
[the remark this answers is not in the document]
Comment (elenatvk) [reply templates/sag-db-mysql/README.md]: I'm sorry for the delay on this, I had overlooked your comments. So, are we updating this example to removing the MWS component - I also updated the description to say "except My webMethods Server"

File: templates/sag-db-mysql/README.md [hunk 4]
```diff
@@ -59,27 +84,26 @@ When importing the composite template in Command Central, you have to attach the
...
+  db.product.scripts=[DatabaseComponentConfigurator,OBEcdc,WOKcdc,TNScdc,MWScdc,B2BCloudCdc,MATcdc,ODEcdc,PIEcdc, PIEEmbeddedCdc, PIEMobileCdc, WMNcdc, WPEcdc, WSTcdc] \
```
[the remark this answers is not in the document]
Comment (elenatvk) [reply templates/sag-db-mysql/README.md]: updated

File: templates/sag-db-mysql/README.md [hunk 5]
```diff
@@ -59,27 +84,26 @@ When importing the composite template in Command Central, you have to attach the
...
 ```
 
-To install Database Component Configurator 10.5 on the Command Central node with alias `local`, create a database named `webm` and a database user named `webm` with password `webm`, and create all database components schemas with user `root` and password `root`:
+To install Database Component Configurator 10.5 on the Command Central node with alias `local`, create a database named `webm` and a database user named `webm` with password `webm`, and create database schemas for all products with user `root` and password `root`:
 
 ```bash
 sagcc exec templates composite apply sag-db-mysql \
   db.version=latest repo.product=products-10.5 repo.fix=fixes-10.5 nodes=local \
   db.host=mysql db.admin.username=root db.admin.password=root \
-  db.STR.components=[STR] \
```
Comment (YanaSimeonova) [inline templates/sag-db-mysql/README.md]: by default in the template the db.STR.components is empty, so either we nned to change the template or we need to keep this in the example

Comment (elenatvk) [reply templates/sag-db-mysql/README.md]: Updated - I added the line back in the readme file.

**Pull request closed without merging**